src/bp-core/profile-search/templates/members/bps-form-default.php

```diff
@@ -130,7 +130,7 @@
 			$km          = __( 'km', 'buddyboss' );
 			$miles       = __( 'miles', 'buddyboss' );
 			$placeholder = __( 'Start typing, then select a location', 'buddyboss' );
-			$icon_url    = plugins_url( 'bp-profile-search/templates/members/locator.png' );
+			$icon_url    = BP_PLATFORM_API . 'bp-core/profile-search/templates/members/locator.png';
```

Comment:
@shanebp it should be a like this `buddypress()->plugin_url` instead of `BP_PLATFORM_API`
```suggestion
			$icon_url    = buddypress()->plugin_url . 'bp-core/profile-search/templates/members/locator.png';
```